src/app/shared/components/capture-receipt/receipt-preview/receipt-preview.component.ts

```
@@ -56,47 +60,84 @@ export class ReceiptPreviewComponent implements OnInit {
     });
     await cropReceiptModal.present();
     this.isCropModalOpen = true;
-    const { data } = await cropReceiptModal.onWillDismiss();
+    const { data }: { data?: { base64ImageWithSource: Image } } = await cropReceiptModal.onWillDismiss();
     this.isCropModalOpen = false;
 
     if (data && data.base64ImageWithSource) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)